src/__spec_helper__/test-utils.js

```diff
@@ -1,12 +1,83 @@
 const isUpper = char => char.toUpperCase() === char;
 const humpToDash = (acc, char) => `${acc}${isUpper(char) ? `-${char.toLowerCase()}` : char}`;
 
-export const toCSSCase = (str) => {
+const toCSSCase = (str) => {
   return str.split('').reduce(humpToDash, '');
 };
 
-export const assertStyleMatch = (styleSpec, component, opts) => {
+const assertStyleMatch = (styleSpec, component, opts) => {
   Object.entries(styleSpec).forEach(([attr, value]) => {
     expect(component).toHaveStyleRule(toCSSCase(attr), value, opts);
   });
 };
+
+const makeArrayKeys = n => [...Array(n).keys()];
+
+const keyPress = (code) => {
+  const ev = new KeyboardEvent('keydown', { which: code });
+  document.dispatchEvent(ev);
+};
+
+const keyMap = {
+  UpArrow: '38',
+  DownArrow: '40',
+  RightArrow: '39',
+  LeftArrow: '37',
+  Enter: '13',
+  Tab: '9'
+};
+
+const repeat = action => (n = 1) => makeArrayKeys(n).forEach(() => action());
```

Comment:
if I understand correctly - you're creating an array to just use `forEach` method? If yes then a `while` loop could be used here.

Reply:
These are the utils @samhinton88 created, I've just lifted them up a level so they can be shared. I'm inclined to leave it as is as it's not the what the aim of the ticket is